app/views/app/decision.html.erb

```diff
@@ -68,15 +68,8 @@
           <p>This offer has been `+offer_status+'.</p>';
         }
       }
-      function postHelper(url, data, action, blob=false){
-        fetch(url, {
-          headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/json; charset=utf-8',
-          },
-          body: JSON.stringify(data),
-          method: 'POST',
-        }).then(function(response) {
+      function fetchHelper(url, init, action, blob=false){
```

Comment:
one line comment saying this is helper factoring out frequently used code around a fetch